indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -521,6 +521,29 @@ def _process(self):
                                                     user_data=self._get_user_data(), management=True)
 
 
+class RHRegistrationCreateMultiple(RHManageRegFormBase):
+    """Create multiple registrations for Indico users (management area)"""
+
+    def _register_user_personal_data_only(self, user, notify):
+        data = {pdt.name: getattr(user, pdt.name, None) for pdt in PersonalDataType}
+        with db.session.no_autoflush:
+            create_registration(self.regform, data, management=True, notify_user=notify)
+
+    def _process(self):
+        form = CreateMultipleRegistrationsForm(regform=self.regform, open_add_user_dialog=request.method == 'GET')
```

Comment:
- get the default value for `notify_user` from the session (and store it on successful submission)
- I'd wrap `request.method == 'GET'` in parentheses. looks pretty awful without them IMO